### ISSUE TITLE
Bulk export/import?

### ISSUE THREAD

**Issue by D3SOL4TE** [subabrain/AbiBmpReplacer#1]
Would it be possible to add the ability to export/import multiple files at once?

**Comment by subabrain** [comment]
yes i already done this some months ago but  i dont have this version anymore ...

i will have a look ...

Regards
Robert

**Comment by D3SOL4TE** [comment]
Thanks.
